<?xml version='1.0' encoding='utf-8'?>
<ui version="4.0"><widget class="QMainWindow" name="MainWindow"><property name="windowTitle"><string>Clientes</string></property><property name="geometry"><rect><x>100</x><y>100</y><width>800</width><height>600</height></rect></property><widget class="QWidget" name="centralwidget"><widget class="QTableWidget" name="tableWidget"><property name="geometry"><rect><x>10</x><y>10</y><width>780</width><height>400</height></rect></property><property name="columnCount"><number>6</number><number>0</number></property><property name="rowCount" /></widget><widget class="QPushButton" name="btnAgregar"><property name="geometry"><rect><x>10</x><y>420</y><width>100</width><height>30</height></rect></property><property name="text"><string>Agregar</string></property></widget><widget class="QPushButton" name="btnEditar"><property name="geometry"><rect><x>120</x><y>420</y><width>100</width><height>30</height></rect></property><property name="text"><string>Editar</string></property></widget><widget class="QPushButton" name="btnEliminar"><property name="geometry"><rect><x>230</x><y>420</y><width>100</width><height>30</height></rect></property><property name="text"><string>Eliminar</string></property></widget><widget class="QPushButton" name="btnActualizar"><property name="geometry"><rect><x>340</x><y>420</y><width>100</width><height>30</height></rect></property><property name="text"><string>Actualizar</string></property></widget></widget><layout class="QVBoxLayout" name="verticalLayout" /><widget class="QMenuBar" name="menubar"><property name="geometry" /></widget><widget class="QStatusBar" name="statusbar" /></widget></ui>